backend/embedding/main.py

```diff
@@ -139,21 +146,35 @@ def main(
     filenames: list[str],
     chunk_size: int,
     chunk_overlap: int,
+    enable_partition_pdf: bool,
 ):
     exec_id = ""
     try:
         exec_id = get_exec_id()
     except Exception as e:
-        print(f"[ERROR] Failed to get exec_id: {e}")
+        logger.info(f"[ERROR] Failed to get exec_id: {e}")
```

Comment:
The error logs should be handled with `logger.error`

Reply:
fixed